src/java/com/phenix/pct/ABLUnit.java

```diff
@@ -92,8 +92,10 @@ public void addConfiguredFileset(FileSet set) {
         testFilesets.add(set);
     }
 
-    private void writeJsonConfigFile() throws IOException {
+    private void writeJsonConfigFile() throws IOException, RuntimeException {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS00112)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the declaration of thrown exception 'java.lang.RuntimeException' which is a runtime exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)